inlong-sort/sort-formats/format-inlongmsg-base/src/main/java/org/apache/inlong/sort/formats/inlongmsg/InLongMsgValidator.java

```diff
@@ -21,17 +21,20 @@
 import org.apache.inlong.sort.formats.base.TableFormatUtils;
 
 import org.apache.flink.table.descriptors.DescriptorProperties;
-import org.apache.flink.table.descriptors.FormatDescriptorValidator;
+import org.apache.flink.table.descriptors.DescriptorValidator;
+
+import static org.apache.inlong.sort.formats.base.TableFormatConstants.FORMAT_PROPERTY_VERSION;
+import static org.apache.inlong.sort.formats.base.TableFormatConstants.FORMAT_TYPE;
 
 /**
  * Validator for mixed inlongmsg formats.
  */
-public class InLongMsgValidator extends FormatDescriptorValidator {
+public class InLongMsgValidator implements DescriptorValidator {
 
     @Override
     public void validate(DescriptorProperties properties) {
-        super.validate(properties);
-
+        properties.validateString(FORMAT_TYPE, false, 1);
```

Comment:
duplicate code with InLongMsgMixedValidator.java

Reply:
Sorry, I don't understand what you mean.

Reply:
InLongMsgMixedValidator have duplicate code with InlongMsgValidator could you please extract these code ?

Reply:
done